src/textual/reactive.py

```diff
@@ -255,16 +233,13 @@ async def await_watcher(awaitable: Awaitable) -> None:
 
         def invoke_watcher(
             watch_function: Callable, old_value: object, value: object
-        ) -> bool:
+        ) -> None:
```

Comment:
return value is unused, and originally was what got me thinking about the bug (the original code stopped iterating over watchers when this returned `False`). Better just get rid of it.